archive/mozilla_mar.ksy

```diff
@@ -0,0 +1,116 @@
+meta:
+  id: mozilla_mar
+  title: Mozilla ARchive
```

Comment:
Probably Firefox can be filled in as a `/meta/application` connected with this format?

Reply:
It is any Mozilla product really.